src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,29 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - According to the checklist, you should prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components to import data. Consider lifting the state up to the parent component and passing the user data as a prop to the `TodoInfo` component.